servers/shareable-lists-api/src/events/utils.ts

```diff
@@ -0,0 +1,40 @@
+import {
```

Comment:
Symmetrical Enum helpers.  Helps us convert from Prisma enums to the EventBridge package enums. If this is used elsewhere, will turn into a package.